### PR TITLE
Bumped api-specification version number

### LINKED ISSUE #184: npm install fails for containerjs-api-electron
When trying to add this to a new project, it fails to install.

This is due to our nasty post-install scripts!

image

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Fixes #184

Also removed some other redundant files

I'm going to do a manual release of `api-specification` to npm to resolve #184 - after that, I'll look into a better process for managing our version numbers!

### CODE REVIEW

File: copy-markdown.js [hunk 1]
```diff
@@ -1,26 +0,0 @@
-const fs = require('fs');
```
Comment (BenLambertNcl) [inline copy-markdown.js:1]: :+1:

**Review by BenLambertNcl — COMMENTED** [review]
Do we not need to remove the `postinstall` script from `api-specification` to fix this issue? That what it seemed to be failing on?

**Comment by ColinEberhardt** [comment]
> Do we not need to remove the postinstall script from api-specification to fix this issue? That what it seemed to be failing on?

It's gone :-)

https://github.com/symphonyoss/ContainerJS/commit/49252349b23970080b6f8bf20890ece0f31cf01a#diff-58f5dfe6b9475d81f90ba1659d5e93b9L8

... and development is so much easier now without them!

**Comment by BenLambertNcl** [comment]
Oops, I was looking on an old branch :-)

**Review by BenLambertNcl — APPROVED** [review]

**Pull request merged**